.github/scripts/ep_hooks.py

```
@@ -0,0 +1,267 @@
+
+class EPHooks:
+    def __init__(self, repo, skills_path, shadow=False,
+                 bot_login="github-actions[bot]",
+                 reviewed_label="rfe-creator-auto-reviewed"):
+        self.repo = repo
+        self.skills_path = skills_path
+        self.shadow = shadow
+        self.bot_login = bot_login
+        self.reviewed_label = reviewed_label
+
+    def _gh(self, args):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_gh`

(ANN202)